crypto/Kconfig

```
@@ -23,6 +23,13 @@ if NRF_CC310_BL
 	default n # Only the no-interrupts version currently verified to work with Zephyr.
 endif
 
+menuconfig NRF_CC3XX_MBEDCRYPTO
+	bool "nrf_cc3xx_mbedcrypto SW crypto library for nRF devices with CryptoCell CC3xx."
```

based on:
> -This will be usefull for the use cases where the nordic security
 backend is not enabled but the mbedcrypto library needs to be linked

this sounds like that other part of the build system needs `NRF_CC3XX_MBEDCRYPTO` but without using nrf security should have the responsibility to select this setting.
Not the end-user, so I suggest to remove the prompt.
```suggestion
	bool
```

---

Agree on this. This prevents direct usage (makes it only possible to set it by Kconfig)